DOCUMENTATION.md

```diff
@@ -708,7 +708,7 @@ If the request is not successful, the client throws a `Notifications::Client::Re
 |:---|:---|:---|:---|
 |`400`|`ValidationError: id is not a valid UUID`|`BadRequestError`|Check the notification ID|
 |`400`|`PDFNotReadyError: PDF not available yet, try again later`|`BadRequestError`|Wait for the notification to finish processing. This usually takes a few seconds|
-|`400`|`BadRequestError: Document did not pass the virus scan`|`BadRequestError`|You cannot retrieve the contents of a letter notification that contains a virus|
+|`400`|`BadRequestError: File did not pass the virus scan`|`BadRequestError`|You cannot retrieve the contents of a letter notification that contains a virus|
```

Comment:
this error message (both for letters and doc dl) was changed in https://github.com/alphagov/notifications-api/pull/2696/commits/a16f79896b9cdc2ec534cb37b784078edf66b17d